README.md

```diff
@@ -1,19 +1,238 @@
-# rfcs
-RFCs for changes to OBS Studio (and supporting toolset)
+# OBS Project RFCs
 
+[OBS Project RFCs]: #obs-project-rfcs
 
-Basic RFC process:
-- Fork the repo
-- Copy `0000-template.md` to `accepted/XXXX-my-rfc-title.md`. Make sure
-  the title is descriptive. RFC number should match the next PR number.
-- Fill out the RFC template. Our template is a bit basic right now, so
-  I recommend looking at the other submitted RFCs for guidance.
-- Submit the RFC as a pull request.
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
 
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the OBS Project 
+community.
 
-Things to keep in mind:
-- An RFC is designed to be an open discussion. Please be accepting to
-  and reponsive to any feedback, even if you do not agree.
-- Not all RFCs will be accepted. Some may require extensive work or
-  deviation from their original submissions.
-- As always, be kind and respectul with your discussions.
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features, functionality, and changes to undergo
+a peer-review and allow all project stakeholders to weigh in their comments
+and ensure the direction is consistent with the vison of the project.
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#obs-project-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+OBS Studio, obs-browser, or any of the other plugins/modules. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - ????
```

Comment:
I think "substantial" changes would either be new features or changing the implementation of an existing feature/module.

Reply:
Agreed, Substantial changes should reflect anything that replaces existing features, the GUI or implementations of things.

Reply:
This can be:
- Anything that adds a major feature
- Anything that changes internal engineering design of any part of the program.  For example, changing the slideshow to be dynamic rather than static would be considered a big engineering design change
- Anything that changes UX/UI design, or changes something that the user has to interact with to a significant extent and it's a big change.
- Anything that changes API, ABI, or settings storage format of the program

There probably can be exceptions to this.  For example, if it's a small change where your work isn't really "wasted" if it's rejected, then it's probably fine to directly PR without an RFC.  Another example is where a change is sort of obviously ideal, like making a pull request to add texture-based encoding for QSV, that is a recent fairly significant change but is sort of "obvious".  Or some things written by maintainers and/or trusted core contributors (although even they are encouraged to use RFCs).  Or things that have already been approved of by the maintainers, and discussion has already happened in some form somewhere else.